.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,84 @@
+name: Bug Report
+description: Help us improve.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
```

Comment:
Does this actually show up in the issue? If so idk if this is appropriate for this.

Reply:
It shows up in the form, but isn't rendered in the issue.